docs/install/manual-install.md

```diff
@@ -128,11 +128,45 @@ As well as installing dependencies, this script compiles our CSS (using
 `bin/make_css`), installs the database schema (using `bin/update-schema`),
 and compiles any translation `.mo` files (using `commonlib/bin/gettext-makemo`).
 
+#### macOS troubleshooting
+
+Some Perl modules may fail to install without certain packages and paths in place.
+
+**Image::PNG::QRCode** requires the `libpng` package.
+
+For any **SSL-related modules**, make sure `openssl` is installed. If using homebrew:
+
+    $ brew install --cask openssl
```

Comment:
This command fails with `Error: Cask 'openssl' is unavailable: No Cask with this name exists.` when I run it locally.

I think it's actually the `openssl@1.1` package that's needed, so perhaps that could be added to the existing `brew install` command on line 68?

Reply:
Yeah, I am a bit confused now - `brew install --cask openssl@1.1` fails too (though I have it pasted in my notes, like I ran it before and it worked - I am not really sure what `--cask` does versus installing normally). I will put `brew install openssl@1.1`, unless you know of a reason this is incorrect?

Reply:
Yeah, I think `brew install openssl@1.1` should do the trick!